packages/components/src/docs/Migration/ui-video.mdx

```diff
@@ -0,0 +1,87 @@
+# Migration Guide: From `ui-video` to `sd-video`
```

Comment:
I would love to see a more detailed description about the sd-video component only provided a wrapper compared to the ui-video providing the whole player. Its clear if you know the component but would be nice to have it clear mentioned in the beginning of the component.